Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbon.cs

```diff
@@ -3101,9 +3101,10 @@ private void OnTabPropertyChanged(object sender, PropertyChangedEventArgs e)
         private void OnRibbonQATButtonsClearing(object sender, EventArgs e)
         {
             // Stop tracking changes in button properties
-            foreach (IQuickAccessToolbarButton qatButton in QATButtons)
+            foreach (var component in QATButtons)
```

Comment:
Convert this into a typed where clause, then foreach over that

Reply:
Reverted to original, added 'todo' comment